index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name = "viewport" content="width=device-width, initial-scale=1">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="headerbackground">
+        <div class="header">
+            <h1 class="Logo">My Logo</h1>
+
+            <nav class="navlinks">
+                 <ul class="navitems">
+                    <li class="vector1"><a href="#"><img src="Images/Vector1.png" alt="shape"></a></li>
+                    <li class="vector2"><a href="#"><img src="Images/Vector2.png" alt="shape"></a></li>
+                    <li class="vector3"><a href="#"><img src="Images/Vector3.png" alt="shape"></a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="headline">
+            <ul class="images">
+                <li class="shape1"><img src="Images/Ellipse8.png" alt="shape"></li>
+                <li class="shape2"><img src="Images/Ellipse11.jpg" alt="shape"></li>
+                <li class="shape3"><img src="Images/Rectangle7.png" alt="shape"></li>
+                <li class="shape4"><img src="Images/Ellipse5.png" alt="shape"></li>
+                <li class="shape5"><img src="Images/Rectangle4.png" alt="shape"></li>
+                <li class="shape6"><img src="Images/Ellipse3.png" alt="shape"></li>
+            </ul>
+
+            <h2 class="header2">I’m Iptisam 
+                Glad to see you!</h2>
+            <p class="Me">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="links">LET’S CONNECT</p>
+            <ul class="images">
```

Comment:
### Your Homepage
![margins-fixable (copy)](https://user-images.githubusercontent.com/93039359/185141259-62792020-40da-4f60-9ad5-d92d338cb8fd.png)

### Figma Homepage
![margins-fix (copy)](https://user-images.githubusercontent.com/93039359/185141293-56bcd1d4-d7c3-4772-a894-f48a01cdac4b.png)

- [ ] Please fix your margins. You can see the margin in the Figma template. Please make your design similar to the Figma template to make it look professional :+1: